core/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -236,6 +236,12 @@ case class ExecutionEvent(
   endTime: DateTime
 )
 
+case class MetadataParams(
+  includeKeys: Seq[String] = Seq.empty[String],
```

Comment:
can you use more concrete type such as `List`, `Vector` or `Set instead of `Seq`? there was a discussion [here](https://broadinstitute.slack.com/archives/C9ZSW4R7U/p1536949814000100)